question-servers/freeformPythonLib/prairielearn_test.py

```diff
@@ -4,19 +4,14 @@
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../lib')))
 
 import prairielearn as pl  # noqa: E402
-import unittest            # noqa: E402
 import lxml.html           # noqa: E402
 
 
-class TestPrairielearnLib(unittest.TestCase):
+class TestPrairielearnLib:
```

Comment:
Wrapping tests in a class is unnecessary with `pytest` - let's make `test_inner_html` a top-level function.